x-pack/plugins/security_solution/public/timelines/components/flyout/pane/pane.tsx

```diff
@@ -24,39 +26,31 @@ const FlyoutPaneComponent: React.FC<FlyoutPaneComponentProps> = ({
   timelineId,
   visible = true,
 }) => {
-  const { euiTheme } = useEuiTheme();
   const ref = useRef<HTMLDivElement>(null);
-  const timeline = useMemo(
-    () => (
-      <StatefulTimeline
-        renderCellValue={DefaultCellRenderer}
-        rowRenderers={defaultRowRenderers}
-        timelineId={timelineId}
-      />
-    ),
-    [timelineId]
-  );
+  const isFullScreen = useShallowEqualSelector(inputsSelectors.timelineFullScreenSelector) ?? false;
+
+  const styles = usePaneStyles();
+  const wrapperClassName = classNames('timeline-wrapper', styles, {
+    'timeline-wrapper--full-screen': isFullScreen,
+    'timeline-wrapper--hidden': !visible,
+  });
 
   return (
     <div data-test-subj="flyout-pane" ref={ref}>
```

Comment:
I figure this is a quick solution to get this fixed now, but would it be more conducive in the long run to replace the `EuiFlyout` references with `EuiModal` in `query_tab_content/index`..and all the other tab contents? Or just outright remove the flyout references?

Reply:
Do you mean stop using `EuiFlyoutHeader`, `EuiFlyoutBody`, and `EuiFlyoutFooter` and start using `EuiModal` components? I don't think that's a direct change. Also, I am not sure this will have any benefit, we made the timeline look like a modal externally, but we are not using `EuiModal` at all.

Reply:
Yea, it was more to just use some standard Eui components for everything, but you're right it would not be a straightforward change. No worries